app/models/stored_value.rb

```diff
@@ -0,0 +1,23 @@
+require 'base64'
+
+##
+# This class represents temporary storage for values processed in background.
+# This is a database solution, wich should not be the only one, as it has many limitations.
+
+class StoredValue < ApplicationRecord
+  ##
+  # Writes the value.
+  # It encodes the value in Base64, to be able to store any data.
+
+  def self.write(result_key, result_value, expire_at: nil)
+    record = self.find_by(key: result_key) || new(key: result_key)
+    record.value = Base64.encode64(result_value)
```

Comment:
is base64 required? how about we use some blob type in DB? encoding and decoding of 16 MB seems unnecessary

Reply:
Moved to binary. Do not know, why I came up with this instead of binary in DB :(